dot/state/block_finalisation.go

```diff
@@ -142,16 +142,20 @@ func (bs *BlockState) SetFinalisedHash(hash common.Hash, round, setID uint64) er
 		bs.notifyFinalized(hash, round, setID)
 	}
 
-	pruned := bs.bt.Prune(hash)
-	for _, hash := range pruned {
-		blockHeader := bs.unfinalisedBlocks.delete(hash)
-		if blockHeader == nil {
-			continue
-		}
+	forkOriginToChain := bs.bt.Prune(hash)
```

Comment:
what means this `forkOriginToChain`?

Reply:
It's a mapping from the fork origin (block hash of the block on the canonical chain where the fork originates from) to the forked chain (slice of block hashes corresponding to the blocks of the forked chain, from lower block number to higher). Would you foresee a clearer name (maybe a bit longer is fine)?